### PR TITLE
Cleans up and dmdoc's implant.dm

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Mostly variable names and dmdoc. Behavior unchanged.

## Why It's Good For The Game
Readability, dmdoc.

## Changelog
Not player-facing.

### CODE REVIEW

File: code/game/objects/items/implants/implant.dm [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 	icon_state = "generic" //Shows up as the action button icon
 	actions_types = list(/datum/action/item_action/hands_free/activate)
-	var/activated = TRUE //1 for implant types that can be activated, 0 for ones that are "always on" like mindshield implants
+	///true for implant types that can be activated, false for ones that are "always on" like mindshield implants
+	var/activated = TRUE
+	///the mob that's implanted with this
 	var/mob/living/imp_in = null
+	///implant color, used for visuals
 	var/implant_color = "b"
+	///if false, upon implantation of a duplicate implant, an attempt to combine the new implant's uses with the old one's uses will be made, deleting the new implant if succesful or stopping the implantation if not
```
Comment (coiax) [inline code/game/objects/items/implants/implant.dm:15]:
```suggestion
	///if false, upon implantation of a duplicate implant, an attempt to combine the new implant's uses with the old one's uses will be made, deleting the new implant if successful or stopping the implantation if not
```

File: code/game/objects/items/implants/implant.dm [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 	icon = 'icons/obj/implants.dmi'
 	icon_state = "generic" //Shows up as the action button icon
 	actions_types = list(/datum/action/item_action/hands_free/activate)
-	var/activated = TRUE //1 for implant types that can be activated, 0 for ones that are "always on" like mindshield implants
+	///true for implant types that can be activated, false for ones that are "always on" like mindshield implants
+	var/activated = TRUE
+	///the mob that's implanted with this
 	var/mob/living/imp_in = null
+	///implant color, used for visuals
 	var/implant_color = "b"
```
Comment (coiax) [inline code/game/objects/items/implants/implant.dm:14]: Could you explain what this `b` actually represents?

File: code/game/objects/items/implants/implant.dm [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
+	///if false, upon implantation of a duplicate implant, an attempt to combine the new implant's uses with the old one's uses will be made, deleting the new implant if succesful or stopping the implantation if not
 	var/allow_multiple = FALSE
+	///uses
 	var/uses = -1
```
Comment (coiax) [inline code/game/objects/items/implants/implant.dm:18]: I am unclear if this documention is actually worth anything.

Comment (interestingusernam3) [reply code/game/objects/items/implants/implant.dm]: improved.

File: code/game/objects/items/implants/implant.dm (hunk 2 above)
Comment (interestingusernam3) [reply code/game/objects/items/implants/implant.dm]: explained.

File: code/game/objects/items/implants/implant.dm (hunk 1 above)
Comment (interestingusernam3) [reply code/game/objects/items/implants/implant.dm]: Commited.

**Pull request merged**